docs/src/writing_good_rules.md

```diff
@@ -132,6 +132,10 @@ Furthermore, AD systems make different trade-offs in performance due to their de
 This means that a certain rule will help one AD system, but not improve (and also not harm) another.
 Below, we list some patterns relevant for the [Zygote.jl](https://github.com/FluxML/Zygote.jl) AD system.
 
+Rules for functions which mutate its arguments, e.g. `sort!`, should not be written at the moment.
+While technically they are supported, they would break [`Zygote.jl`](https://github.com/FluxML/Zygote.jl) so that [it would sometimes quietly return the wrong answer](https://github.com/JuliaDiff/ChainRulesCore.jl/issues/242).
```

Comment:
```suggestion
While technically they are supported, they would break [Zygote.jl](https://github.com/FluxML/Zygote.jl) such that [it would sometimes quietly return the wrong answer](https://github.com/JuliaDiff/ChainRulesCore.jl/issues/242).
```